### ISSUE TITLE
Gradient clipping is no longer recommended?

### ISSUE THREAD

**Issue by cloudygoose** [huggingface/transformers#24677]
### System Info

Hi,

I just found that in the current examples (e.g., https://github.com/huggingface/transformers/blob/main/examples/pytorch/language-modeling/run_clm_no_trainer.py), gradient clipping is no longer applied. Is there any particular reason? Is it okay if we add a line to do gradient clipping myself?

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

N/A

### Expected behavior

N/A

**Comment by ydshieh** [comment]
You can definitely experiment with gradient clipping.

The `transformers` GitHub pages are reserved for issues or feature requests. The question here is not in the scope, and [[Hugging Face Forums](https://discuss.huggingface.co/)](https://discuss.huggingface.co/) is a better place, if you have further question on this topic.
